applications/GeoMechanicsApplication/custom_utilities/equation_of_motion_utilities.cpp

```diff
@@ -58,4 +58,12 @@ Vector GeoEquationOfMotionUtilities::CalculateDetJsInitialConfiguration(const Ge
     return det_Js_initial_configuration;
 }
 
+Matrix GeoEquationOfMotionUtilities::CalculateDampingMatrix(double        RayleighAlpha,
+                                                            double        RayleighBeta,
+                                                            const Matrix& rMassMatrix,
+                                                            const Matrix& rStiffnessMatrix)
+{
+    return rMassMatrix * RayleighAlpha + rStiffnessMatrix * RayleighBeta;
```

Comment:
Correct, but the formula is usually written as: alpha*M + beta*K
Please do the same here.

Is there a need for checking the sizes of M and K before doing this operation, or do we accept that will be o.k. as those are generated for the same element and therefore have the same and correct size.

Reply:
Thank you. Indeed ublas provides both options. Changed it. Yes the same size. In all three cases the matrices are generated just before CalculateDampingMatrix function.